assets/src/app/pipes/job_type.pipe.ts

```diff
@@ -16,6 +16,7 @@ export class JobTypePipe implements PipeTransform {
       { id: 'ftp_order', name: 'FTP transfer' },
       { id: 'generate_dash', name: 'Generate DASH' },
       { id: 'gpac_dash', name: 'DASH generation' },
+      { id: 'set_language', name: 'Language setting' },
```

Comment:
you miss to set the icon too

Reply:
Icon added in 951764fb2806b04cff3747e783d8769460282991